app/build.gradle

```diff
@@ -187,6 +184,34 @@ dependencies {
     androidTestCompile 'com.android.support:support-annotations:25.1.0'
 }
 
+// SHA and timestamp caching courtesy of https://github.com/gdg-x/frisbee/blob/develop/app/build.gradle#L193-L218
+def commitSha() {
+    def file = new File(project.buildDir, "commit-sha.txt")
+    if (file.exists()) {
+        return file.text
+    }
+
+    def value = 'git rev-parse --short HEAD'.execute([], project.rootDir).text.trim()
```

Comment:
Executing git like this will most probably break in windows machines. Please check. 
That's why we used [grgit][https://github.com/ajoberstar/gradle-git/] in frisbee. I believe it is also more readable like this `getHeadGitCommit().abbreviatedId`